st2common/st2common/util/api.py

```diff
@@ -53,3 +55,19 @@ def get_full_public_api_url(api_version=DEFAULT_API_VERSION):
     api_url = get_base_public_api_url()
     api_url = '%s/%s' % (api_url, api_version)
     return api_url
+
+
+def get_mistral_api_url(api_version=DEFAULT_API_VERSION):
+    """
+    Return a URL which Mistral uses to talk back to the StackStorm API.
+
+    Note: If not provided it defaults to the public API url.
+    """
+    if cfg.CONF.mistral.api_url:
+        api_url = get_url_without_trailing_slash(cfg.CONF.mistral.api_url)
+        api_url = '%s/%s' % (api_url, api_version)
+    else:
+        LOG.warn('"mistral.api_url" not set, using auth.api_url')
```

Comment:
Nope, not a typo.

That's what it's using now and this value acts as a single source of truth for public API URL.